src/Games/NexusMods.Games.Larian/BaldursGate3/Pipelines.cs

```diff
@@ -49,10 +48,10 @@ public static IServiceCollection AddPipelines(this IServiceCollection serviceCol
                     }
                 }
             )
-            .UseCache(
-                keyGenerator: static hash => hash,
+            .StoreInMemory(
+                keySelector: static hash => hash,
                 keyComparer: EqualityComparer<Hash>.Default,
-                capacityPartition: new FavorWarmPartition(totalCapacity: 300)
+                shouldDeleteKey: async (tuple, _) => await fileStore.HaveFile(tuple.Item1)
```

Comment:
I'm not sure I follow this.

I was thinking this should be something like:
```csharp
shouldDeleteKey: (tuple, _) => LoadoutFile.FindByHash(db, hash).Count == 0;
```

Remove if there are no loadout items that with this hash, we can have them in the library, but we don't care about those. So if the user removes a bunch of mods or deletes a loadout or something like that, all those entries can get evicted.